test/Garnet.test/RespTests.cs

```diff
@@ -3539,5 +3539,192 @@ private static void AssertExpectedClientFields(string list)
             static void AssertField(string line, string[] fields, string name)
             => ClassicAssert.AreEqual(1, fields.Count(f => f.StartsWith($"{name}=")), $"In {line}, expected single field {name}");
         }
+
+        #region GETEX
+
+        [Test]
+        public void GetEXBasicTestWithSERedisApi()
```

Comment:
Please rename tests to GetExpiry... just for consistency please :)

Reply:
Fixed